index.html

```diff
@@ -1,75 +1,77 @@
-                    <a href="" class="">Learn more</a>
-                </div>
-                <form class="inputs">
-                    <fieldset>
-                        <label>Email address</label>
-                        <input type="email">   
-                    </fieldset>
-                    <fieldset>
-                        <label>Phone <span class="widetype">(recommended)</span></label>
-                        <input type="text"> 
-                        <span class="small">Standard call, messaging or data rates may apply.</span>  
-                    </fieldset>
-                    <fieldset>
-                        <label>Password</label>
-                        <div class="lock-right"><input type="password" ><img src="images/lock-right.png" alt="Lock icon for the password input"></div>
+<head>
+    <title>HTML Forms Project</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="css/styles.css" rel="stylesheet">
+</head>
 
-                    </fieldset>
-                    <button><span class="lock"></span> Create Account</button>
-                </form>
-                <p class="terms">
-                    By clicking Create account, you agree to our <a href="#">Terms</a> and have read and acknowledge our <a href="#">Privacy Statement.</a>
-                </p>
-                <div class="captcha">
-                    Invisible reCAPTCHA by Google <a href="#">Privacy Policy</a> and <a href="#">Terms of Use</a>.
-                </div>
-            </div>
-            
+<body>
+    <div class="header">
+        <div class="account">
+            <p>Already have an Intuit account?</p>
+            <br />
+            <a href="#">I forgot my User ID or Password</a>
+        </div>
+        <button><span class="lock"></span> Sign In</button>
+    </div>
+    <div class="center">
+        <div class="logo">
+            <img src="images/logo-intuit.png" alt="Logo">
         </div>
-        <div class="footer">
-            <a href="#">Legal</a><a href="#">Privacy</a><a href="#">Security</a>
-            <p>            
-            © 2020 Intuit, Inc. All rights reserved. Intuit, QuickBooks, QB, TurboTax, ProConnect and Mint are registered trademarks of Intuit Inc.
+        <div class="icons">
+            <ul>
+                <li class="mint-icon"></li>
+                <li class="mint">mint</li>
+                <li class="qbooks-icon"></li>
+                <li class="qbooks">quickbooks</li>
+                <li class="turbo-icon"></li>
+                <li class="turbo">turbotax</li>
+            </ul>
+        </div>
+        <div class="form">
+            <div class="form-title">
+                <p>Create an Intuit account</p>
+                <p class="more">One account for everything Intuit, including Mint. </p>
+                <a href="" class="">Learn more</a>
```

Comment:
It's better to delete empty class. `href` is missing `#`.

Reply:
fixed, thanks